lib/watir/elements/select.rb

```diff
@@ -54,9 +62,16 @@ def select_all(*str_or_rx)
     # @raise [Watir::Exception::NoValueFoundException] if the value does not exist.
     #
 
-    def select!(*str_or_rx)
-      results = str_or_rx.flatten.map { |v| select_by!(v, :single) }
-      results.first
+    def select!(*both, text: nil, value: nil, label: nil)
+      selection = {both: both, text: text, value: value, label: label}.select do |_k, v|
```

Comment:
`#select` and `#select!` should always accept the same types of arguments, so it'd be nice to extract this to be shared code.

Reply:
The refactor for Watir 7 is slightly different since we need to support single, multiple and normal. This was going to be the easy/working copy/paste in the meantime.